backend/src/controllers/company/job/index.ts

```diff
@@ -0,0 +1,52 @@
+import { Request, Response } from 'express';
+import db from '../../../config/db-config';
+
+export const addJobByCompany = async (req: Request, res: Response) => {
+  try {
+    const {
+      title,
+      title_description,
+      description,
+      skills,
+      salary,
+      job_type,
+      experience,
+      is_remotework,
+      requirement,
+    } = req.body;
+
+    const company_id = req.params.company_id;
+    // Insert the new job into the database
+    const insertedJob = await db('jobs')
+      .insert({
+        title,
+        title_description,
+        company_id,
+        description,
+        skills,
+        salary,
+        job_type,
+        experience,
+        is_remotework,
+        requirement,
+      })
+      .returning('id');
+
+    const jobId = insertedJob[0]?.id;
+
+    if (jobId) {
+      const insertedJobDetails = await db('jobs').where('id', jobId).first();
+
```

Comment:
Can't you do simply .returning('*'), so that after the check you would not need to search for it in the DB once again?

Reply:
yes sure.. i changed the code. Thank you.